src/test/java/org/candlepin/resource/EnvironmentResourceTest.java

```diff
@@ -172,7 +169,7 @@ void nothingToDelete() {
     @Test
     void canDeleteEmptyEnvironment() {
         when(this.envCurator.get(anyString())).thenReturn(this.environment1);
-        List<Consumer> mockedQuery = mockedQueryOf().list();
+        List<Consumer> mockedQuery = List.of();
```

Comment:
The `mockedQuery` variable name is a little misleading since it is just a list now that you removed the `mockedQueryOf()`. You could move this to line 174 and do `.thenReturn(List.of())` or rename the variable. This also applies to the other test changes you made.